### ISSUE TITLE
Relegate much of bain documentation to a vignette

### ISSUE THREAD

**Issue by cjvanlissa** [cjvanlissa/bain#8]
The bain help documentation is very extensive, but typically, documentation only consists of the essential information needed to run the function, and a few simple examples for automated testing.

Text as extensive as the current documentation is better suited for a Vignette (which is also automatically included with the package).

CJ will create a bare-bones vignette.
Herbert, can you edit both documents so the right info ends up in each of them?

**Comment by herberthoijtink** [comment]
Ga ik morgen doen.

Ik haal bain van jouw github. Ik update beide files. Ik stuur ze naar mijn github en stuur jou een review request. Right!?

Sincerely,

Herbert Hoijtink

Personal Webpage
http://www.uu.nl/staff/hhoijtink/0

Research Master Methods and Statistics for the Behavioural, Biomedical and Social Sciences
http://www.uu.nl/masters/en/methodology-and-statistics-behavioural-biomedical-and-social-sciences

Minor Methodology and Statistics in the Social and Behavioural Sciences
www.uu.nl/minors/ms<http://www.uu.nl/minors/ms>

Op 14 jan. 2019 om 15:43 heeft C. J. van Lissa <notifications@github.com<mailto:notifications@github.com>> het volgende geschreven:


Assigned #8<https://github.com/cjvanlissa/bain/issues/8> to @herberthoijtink<https://github.com/herberthoijtink>.

**Comment by cjvanlissa** [comment]
Ja, ik push vanavond een ruw opgemaakte versie van hoe de vignette eruit zou kunne zien!

Dr. Caspar J. van Lissa
Assistant professor of developmental data science
Utrecht University, dept. Methodology & Statistics
________________________________
